packages/stores/src/stores/FeatureFlagsStore.ts

```diff
@@ -1,10 +1,7 @@
 import BaseStore from './BaseStore';
 
 const FLAGS = {
-    foo: false,
-    bar: false,
-    baz: false,
-    // Add your flag here 🚀
+    next_wallet: false,
```

Comment:
Add `is_next_wallet_enabled` flag to control the visibility of the `@deriv/wallets`.